lib/index.js

```diff
@@ -2258,6 +2258,28 @@ exposeMethod('uniqBy');
  */
 
 Stream.prototype.uniq = function () {
+    if (!_.isUndefined(typeof Set)) {
```

Comment:
`typeof Set` is never going to be `undefined` (it might be `'undefined'`), so this block will always run.